Patch 1 (#1881)
dzikoysk/reposilite: reposilite-site/data/guides/features/dashboard.md

title: Dashboard
---

Reposilite exposes a frontend that is mounted to `/#/` as [SPA (_single-page application_)](https://en.wikipedia.org/wiki/Single-page_application) in [Vue 3](https://vuejs.org/). You can disable the frontend in [shared configuration](/guide/settings#shared-configuration) if you want to disable an easy way to query your repository or to provide a custom frontend implementation using [static files](/guide/static-files).

#### Browser

> Browser displays the content of the currently selected directory in respect to the provided authentication credentials. For users not logged in it lists only public files, for authenticated users it also displays files that the user has access to.

Reviewer: ```suggestion
Browser displays the contents of the currently selected directory in respect to the provided authentication credentials. For unauthenticated users, only public files will be listed. For authenticated users, all files that the user has access to will displayed.
```

I think these changes would help make this sentence clearer and more consistent.

Author: With your suggestion there's a change of time in the paragraph: "Browser displays" – "will be listed" – "will [be] displayed". 

Change of "not logged in" to "unauthenticated" is a good idea.

![Dashboard Browser Preview](/images/guides/dashboard-browser-preview.png)

> You can control the way how Reposilite displays directory content by opening the adjustments view via the icon in the top-right corner:

Reviewer: ```suggestion
You can control how Reposilite displays directory content by opening the adjustments view via the icon in the top-right corner:
```

I'm not sure `the way how` is the best way of wording this, maybe just `how` would work better?

Author: You're right: https://ell.stackexchange.com/q/246368/80364. I let me confuse myself.

![Browser Adjustments](/images/guides/dashboard-browser-adjustments.png)

#### Console

CLI (_command-line interface_) displays the current Reposilite output and allows to perform available [commands](/guide/standalone#interactive-cli). Only access tokens with management permission can access this view.

![CLI Preview](/images/guides/dashboard-console.png)

You can also filter those messages using filters, but keep in mind that it searches only through the currently cached log entries, not the whole history.

#### Shared configuration
Shared configuration has been already mentioned in:

* [Guide / Setup :: Shared configuration](/guide/settings#shared-configuration)

> Currently, configuration is provided in [CDN (_change detection and Notification_)](https://en.wikipedia.org/wiki/Change_detection_and_notification) format:

Reviewer: I think `CDN format` is referring to [dzikoysk/cdn](https://github.com/dzikoysk/cdn) (`https://github.com/dzikoysk/cdn`), if so, would it be better to link to that instead?

Author: You're right. Wasn't aware of that @dzikoysk developed his own CDN.
